app/views/trips/_form.html.erb

```diff
@@ -1,8 +1,8 @@
 <%= simple_form_for @trip do |f| %>
   <%= f.input :name %>
-  <%= f.input :start_date, as: :date, start_year: 2000,
-    end_year: Date.today.year + 5 %>
-  <%= f.input :end_date, as: :date, start_year: Date.today.year + 10,
+  <%= f.input :start_date, as: :date, start_year: Date.today.year,
```

Comment:
nie możemy zrobić czegoś takiego. Jeśli zechcemy dodać tripa z przeszłości nie będzie takiej możliwości, chyba, że tak ma być

[link](https://github.com/netguru-code-college/vol3-orange/issues/34#issue-354359411)

Reply:
No, mój zamysł jest taki żeby dodawać tylko przyszłe- bo po co ktoś miałby dodawać przeszłe? A im więcej pól do wybrania tym mniej przyjazne dla użytkownika to się robi.

Reply:
Pól byłoby tyle samo, tyle że można by wybrać daty z przeszłości, ale jeśli to ma być do planowania a nie kolekcjonowania tripów to ok ;) można merdżować 🗡